action.yml

```diff
@@ -1,5 +1,4 @@
 name: 'Release with Changelog'
-icon: 'tag'
```

Comment:
Unrelated change

Reply:
I removed it because it causes error – _Property icon is not allowed._
So, should I put it back?

Reply:
You’re right. It’s a duplicate of branding.icon. It’s fine as is

Reply:
OK